src/constants/apiPaths.ts

```diff
@@ -1,9 +1,8 @@
-
 const API_PATHS = {
   product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
```

Comment:
If update other API paths as well, this page https://d3l9zqe2490dzq.cloudfront.net/admin/products and some others should show products.

Reply:
I think that this endpoint will be used in the future tasks, so i don't touch they